ui/tests/test-helper.js

```diff
@@ -11,11 +11,22 @@ import { setup } from 'qunit-dom';
 import { start } from 'ember-qunit';
 import './helpers/flash-message';
 import preloadAssets from 'ember-asset-loader/test-support/preload-assets';
+import { setupGlobalA11yHooks, setRunOptions } from 'ember-a11y-testing/test-support';
 import manifest from 'vault/config/asset-manifest';
 
 preloadAssets(manifest).then(() => {
   setup(QUnit.assert);
   setApplication(Application.create(config.APP));
+  setupGlobalA11yHooks(() => true, {
+    helpers: ['render'],
```

Comment:
Setting the tests to run only on `render` until we fix some widespread issues so that they generally don't run on acceptance tests